e2e/tests/assets/hello-microshift.yaml

```diff
@@ -8,9 +8,7 @@ spec:
   terminationGracePeriodSeconds: 0
   containers:
   - name: hello-microshift
```

Comment:
Should we change all `hello-microshift` to `microshift-test` ? (pod, svc, route etc.)

Reply:
Probably, yes, but that also requires updates on the CI side because we should also update the file name to be consistent. We could do this in 2 steps not to break the CI:
- Create a fixed copy of hello-microshift.yaml called microshift-test.yaml
- Start using it in the CI
- Delete the hello-microshift.yaml in a separate PR

Reply:
I don't think there's a need for that. e2e runs directly from this repository, so if you rename the file and change references in tests that use it, then we're good

Reply:
Okay, that sounds good. I've renamed the file and changed the references in the test.